ONSdigital/census-rm-toolbox
# 1183 am file new

# Motivation and Context

To be able to add addresses confirmed by AR as new cases, a new bulk processor needs to be created to create all the new cases we're going to be getting. The new cases should send a message to Field and RH and it doesn't create any uac-qids for the new cases.

# What has changed

- Added new bulk processor for creating new addresses from AR
- Added new validators for new addresses schema

# How to test?



- To test, run with other branches on pr (case-processor, terraform and kubernetes). Once the new bucket is created, you should be able to upload a file in the bucket and run the new bulk processor. This should create new cases and send them to field and RH. Also check to see to make sure no uac-qids are created from them.

# Links



[Trello](https://trello.com/c/ATLrBzMk)
[Kubernetes](https://github.com/ONSdigital/census-rm-kubernetes/pull/288)
[case-processor](https://github.com/ONSdigital/census-rm-case-processor/pull/181)
[Terraform](https://github.com/ONSdigital/census-rm-terraform/pull/186)
